.github/workflows/docathon-assign.yml

```diff
@@ -0,0 +1,40 @@
+name: Assign User on Comment
+
+on:
+  workflow_dispatch:
+  issue_comment:
+    types: [created]
+
+jobs:
+  assign:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Install Dependencies
+        uses: actions/setup-node@v3
+        with:
+          node-version: '18'
+      - name: Install @octokit/core
+        run: |
+          npm i @octokit/core @octokit/rest
+      - name: Check for "/assigntome" in comment
+        uses: actions/github-script@v4
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        with:
+          script: |
+            const issueComment = context.payload.comment.body;
+            const assignRegex = /\/assigntome/i;
+            if (assignRegex.test(issueComment)) {
+              const assignee = context.payload.comment.user.login;
+              const issueNumber = context.payload.issue.number;
+              const { Octokit } = require("@octokit/rest");
+              const octokit = new Octokit({
+                auth: process.env.GITHUB_TOKEN,
+              });
+              octokit.issues.addAssignees({
```

Comment:
Should we check that issue has not been assigned to others. Also, perhaps it'll be great to only allow that on issue with specific labels

Reply:
Added a check that verifies that the issue is unassigned and if it assigned adds a comment. Also added a check to verify that the label is "docathon-h1-2023" and if not post a comment to pick another issue.